webapi/src/main/scala/org/knora/webapi/messages/v2/responder/resourcemessages/ResourceMessagesV2.scala

```diff
@@ -897,6 +896,89 @@ object UpdateResourceMetadataRequestV2 extends KnoraJsonLDRequestReaderV2[Update
   }
 }
 
+/**
+  * Represents a response after updating a resource's metadata.
+  *
+  * @param resourceIri               the IRI of the resource.
+  * @param resourceClassIri          the IRI of the resource class.
+  * @param maybeLastModificationDate the resource's last modification date, if any.
```

Comment:
I think `lastModificationDate` should always be returned because when we edit the resource's metadata it will always have a `lastModificationDate`. In the GUI it's important to have this information. Otherwise it's not possible to update one field after the other.

Reply:
I agree, I changed it: the value `newModificationDate` is now taken instead of `maybeNewModificationDate`. `newModificationDate` is an `Instant` and not an `Option` and is never empty. If no `maybeNewModificationDate` was given with the request, `Instant.now` is taken as `newModificationDate`. I renamed our parameter accordingly. Let's see if the tests pass through.